wavedromtikz.py

```diff
@@ -813,7 +813,13 @@ def render_help_lines(wavedrom):
 
 
 def render_wavedrom(wavedrom):
-    return r"%s%s%s" % (TIKZ_HEADER,
+    wavedromString = TIKZ_HEADER
+    try:
+        hscale = wavedrom["config"]["hscale"]
+    except KeyError:
+        hscale = 1
+    wavedromString = wavedromString.replace("wavewidth}{1em}", "wavewidth}" + "{" + str(hscale) + "em}")
```

Comment:
This should work for now.
The better solution is to make the `TIKZ_HEADER` a template, instead of replace strings in it.
But I think it's outside the scope of this pr (but hey.. if you want do it.. I'm not here to stop you :P), so I'll accept this for the time being.